MNE/rtProcessing/rthpis.h

```diff
@@ -104,14 +104,42 @@ struct FittingResult {
     QVector<double> errorDistances;
 };
 
+class RtHPISWorker : public QObject
+{
+    Q_OBJECT
+
+//=============================================================================================================
+/**
+* Real-time HPI worker.
+*
+* @brief Real-time HPI worker.
+*/
+public slots:
```

Comment:
if you use new signal slot syntax - slots do not have to be declared anymore:
public: instead of public slots: